readme.md

```diff
@@ -213,6 +213,10 @@
 - [Introduction to Prolog](https://www.youtube.com/watch?v=GHLfeGN5OMk) - Hour-long introduction to Prolog.
 - [Programming In Prolog](https://youtu.be/gJOZZvYijqk) - Four-part video introduction.
 
+### Free Courses
+- [Prolog Online Training](https://www.youtube.com/playlist?list=PLWPirh4EWFpFLjsd3IUqymnCVBaz1Yagg) - Tutorials Point's series on Prolog
```

Comment:
Missing a fullstop here, other than that looks good! Thank you for your contribution!